### DISCUSSION TITLE
Remove filters when using back button

### DISCUSSION THREAD

**Discussion by Jazylucky** [deckerst/aves#673] [Ideas]
It would be very convenient that after you filtered your photos by a certain tag you could remove the tag by using the back button on your Nav Bar. Currently it will exit the app. 

By the way very nice app.

**Comment by deckerst** [comment]
Thanks, glad you like the app.

As for your request, [navigation is a tricky topic](https://github.com/deckerst/aves/issues/550#issuecomment-1465143971) and I will not implement what you suggest.

Filters can be added from many places (via Search, Stats, Info pages), you can land on the collection page with multiple filters (via shortcuts), and you can also land there with a filter via the Albums, Countries, Tags pages. In case of Albums et al., pressing back will bring the user back to the Albums, but that's the exception.

In general, it's easy to remove a filter by tapping on it, independent of navigation.
